plugins/modules/identity/keycloak/keycloak_user.py

```diff
@@ -0,0 +1,530 @@
+            - Username of the user.
+            - This parameter is required only when creating or updating the user.
+
+    realm:
+        type: str
+        description:
+            - They Keycloak realm under which this user resides.
+        default: 'master'
+
+    access:
+        type: dict
+        description:
+            - The access.
+    attributes:
+        type: dict
+        description:
+            - A dict of key/value pairs to set as custom attributes for the user.
+            - Values may be single values (e.g. a string) or a list of strings.
+    client_consents:
+        description:
+            - The client consents.
+        aliases:
+            - clientConsents
+        type: list
+        elements: dict
+        suboptions:
+            client_id:
+                description:
+                    - The client ID.
+                aliases:
+                    - clientId
+                type: str
+            granted_client_scopes:
+                description:
+                    - The granted client scopes.
+                aliases:
+                    - grantedClientScopes
+                type: list
+                elements: str
+    credentials:
+        description:
+            - The credentials.
+        type: list
+        elements: dict
+        suboptions:
+            priority:
+                description:
+                    - The priority.
+                type: int
+            type:
+                description:
+                    - The type.
+                type: str
+            temporary:
+                description:
+                    - The temporary.
+                type: bool
+            value:
+                description:
+                    - The value.
+                type: str
+    disableable_credential_types:
+        description:
+            - The disableable credential types.
+        aliases:
+            - disableableCredentialTypes
+        type: list
+        elements: str
+    email:
+        description:
+            - The email.
+        type: str
+    email_verified:
+        description:
+            - The email verified.
+        aliases:
+            - emailVerified
+        type: bool
+    enabled:
+        description:
+            - The enabled.
+        type: bool
+    first_name:
+        description:
+            - The first name.
+        aliases:
+            - firstName
+        type: str
+    groups:
+        description:
+            - The group pathes.
+        type: list
+        elements: str
+    id:
+        description:
+            - The unique identifier for this user.
+            - This parameter is not required for updating or deleting a user but
+              providing it will reduce the number of API calls required.
+        type: str
+    last_name:
+        description:
+            - The last name.
+        aliases:
+            - lastName
+        type: str
+    not_before:
+        description:
+            - The not before.
```

Comment:
This is probably a timestamp (unix timestamp?). Some more information would be helpful for users.

Reply:
I think this is for revoking tokens before a specific time